experiments/pages/docusign/autoPopulationTutorial.md

```diff
@@ -0,0 +1,135 @@
+
+
+## DocuSign Auto-Populate Integration Set-up
+ 
+##### What you’ll need: 
+1.	A DocuSign sandbox account (create one here: https://go.docusign.com/trial/).
+2.	A Skuid page with data you want to push to a DocuSign template.
+3.	(Eventually) admin access to the production DocuSign account that will be sending the template document for signatures.
+
+##### DocuSign : 
+1.	In your sandbox account, navigate to the admin section. 
+2.	From the menu on the left, select “API and Keys”.
+3.	Click “Add Integrator Key”, define a description and save
+4.	Select your new integrator key and create the correct Skuid Redirect URIs (generate yours with this tutorial: https://docs.skuid.com/latest/en/data/callback-urls-redirect-uris.html)
+5.	Generate a secret key and store it somewhere safe. (Warning: you will only be able to access it one time). Click Save. 
+6.	Go back to DocuSign 
+7.	Create a template and upload the document that is to be populated and sent 
+8.	Define recipients roles and signing order. (If the recipient is going to be variable, only fill in the role name)
+9.	Create signature tabs and define a role for each tab. 
+10.	Create plain text tabs (fields) for each data item that you want to populate.(Make sure to specify the recipient). 
+a.	Define a custom data label (no spaces) for each tab. 
+11.	Take note of the Template Id (the Id can be found in the URL when viewing the document)
+ 
+
+##### Skuid Configure: 
+1.	Go to Data Sources > Authentication Providers
+2.	Create a new Authentication Provider with a unique name and DocuSign Provider type. 
+3.	Set the Authorize Endpoint URL and Token Endpoint URL to:
+a.	 Authorize: https://account-d.docusign.com/oauth/auth
```

Comment:
Just so i understand, do the Authorize URL and Token URL that we have on the DocuSign provider right now only work for Production accounts? i.e. is the `-d` necessary for Sandbox accounts? If so... hmm, i wonder if we should have a separate "DocuSign (Sandbox)" provider type option...

Reply:
Yes, sandbox accounts use -d prepended urls, so that might be helpful to provide that automatically.